helpers/finalizers.go

```diff
@@ -18,3 +18,6 @@ package helpers
 
 // IntegrationPipelineRunFinalizer is the finalizer name to be added to the Integration PipelineRuns
 const IntegrationPipelineRunFinalizer string = "test.appstudio.openshift.io/pipelinerun"
+
+// BuildPipeineRunFinalizer is the finalizer name to be added to Build PipeineRuns
+const BuildPipelineRunFinalizer string = "build.appstudio.openshift.io/pipelinerun"
```

Comment:
Shouldn't this represent that integration service added finalizer? Something like `test.appstudio.openshift.io/SMTH`

Reply:
I'm not sure, I added the label that was suggested in the ticket.  I'm not familiar with labeling conventions in RHTAP.  @dirgim since you created the ticket do you have any input?